configs/config_secret_local_file.json

```diff
@@ -0,0 +1,6 @@
+{
+  "db-user-pass": {
+    "username": "devuser",
+    "password": "S!S*d$zDsb="
+  }
```

Comment:
This file name looks like a Layotto config file while it's not.
To avoid misunderstanding, can you move it to a different package or rename it with some other name ?

Reply:
done, move to another pakage